web/packages/teleterm/src/ui/TopBar/Identity/IdentityList/IdentityList.tsx

```diff
@@ -147,21 +177,6 @@ function DeviceTrustMessage(props: { status: DeviceTrustStatus }) {
   }
 }
 
-// Hack - for some reason xterm.js doesn't allow moving a focus to the Identity popover
-// when it is focused using element.focus(). Moreover, it looks like this solution has a benefit
-// of returning the focus to the previously focused element when popover is closed.
-const focusGrabber = (
```

Comment:
Now going from a terminal to the identity selector with `Cmd + I` doesn't restore focus, does it?

Edit: Nvm, it doesn't work in 17.2.2 either. Could you update the comment in `web/packages/teleterm/src/ui/TopBar/Identity/Identity.tsx` then?

Reply:
Hmm yeah, restoring focus doesn't seem to work. I changed the comment.